pints/functionaltests/differential_evolution_mcmc.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python3
+#
+# Functional tests for DifferentialEvolutionMCMC
+#
+# This file is part of PINTS (https://github.com/pints-team/pints/) which is
+# released under the BSD 3-clause license. See accompanying LICENSE.md for
+# copyright notice and full license details.
+#
+import pints
+import pints.functionaltests as ft
+
+
+def two_dim_gaussian(n_iterations=10000):
+    """
+    Tests :class:`pints.DifferentialEvolutionMCMC`
+    on a two-dimensional Gaussian distribution with true solution
+    ``[0, 0]`` and returns a dictionary with entries ``kld`` and ``mean-ess``.
+
+    For details of the solved problem, see
+    :class:`pints.functionaltests.RunMcmcMethodOnTwoDimGaussian`.
+    """
+    n_warmup = 1000
+    if n_warmup > n_iterations // 2:
+        n_warmup = n_iterations // 10
```

Comment:
I'm not sure I understand this bit?

Reply:
We let people set n_iterations, but not n_warmup, so there needs to be some mechanism to stop n_warmup from being way too big.

a // b is just division with an integer result (rounds towards zero, I think)